python/ray/dashboard/client/src/pages/dashboard/memory/Memory.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  createStyles,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Theme,
+  withStyles,
+  WithStyles,
+  Button,
+} from "@material-ui/core";
+import React from "react";
+import { StoreState } from "../../../store";
+import { connect } from "react-redux";
+import MemoryRowGroup from "./MemoryRowGroup";
+import { dashboardActions } from "../state";
+
+const styles = (theme: Theme) =>
+  createStyles({
+    table: {
+      marginTop: theme.spacing(1),
+    },
+    cell: {
+      padding: theme.spacing(1),
+      textAlign: "center",
+      "&:last-child": {
+        paddingRight: theme.spacing(1),
+      },
+    },
+  });
+
+const mapStateToProps = (state: StoreState) => ({
+  tab: state.dashboard.tab,
+  memoryTable: state.dashboard.memoryTable,
+});
+
+const mapDispatchToProps = dashboardActions;
+
+type State = {
+  // If memory table is captured, it should stop reneweing memory table.
+  memoryTableCapture: boolean;
+};
+
+class MemoryInfo extends React.Component<
+  WithStyles<typeof styles> &
+    ReturnType<typeof mapStateToProps> &
+    typeof mapDispatchToProps,
+  State
+> {
+  state: State = {
+    memoryTableCapture: false,
+  };
+
+  handleMemoryTableCapture = () => {
+    this.setState((state) => ({
+      memoryTableCapture: !state.memoryTableCapture,
+    }));
+    this.props.setShouldObtainMemoryTable(this.state.memoryTableCapture);
```

Comment:
seems like `memoryTableCapture` and `shouldObtainMemoryTable` should always be in sync. Can we get rid of `memoryTableCapture` and just use `shouldObtainMemoryTable` instead?